### PR TITLE
IO tests and bugfix

### PR DESCRIPTION
**Pull request by VikingScientist**

### CODE REVIEW

File: splipy/IO/svg.py [hunk 1]
```diff
@@ -7,6 +7,15 @@
 import re
 from .master import MasterIO
 
+def read_number_and_unit(mystring):
+    try:
+        for i in range(1, len(mystring)):
+            number=float(mystring[:i])
+    except ValueError:
+        unit=mystring[i-1:]
+    return (number, unit)
```
Comment (TheBB) [inline splipy/IO/svg.py:16]: I see you subscribe to the brute force school of problem solving. :smile:

Comment (VikingScientist) [reply splipy/IO/svg.py:16]: If it works it ain't broke 😉  I found that any clever regexp nastyness is always going to fail for wierd float formatting such as [+-.,e] or similar, so being a non-performance-critical part of the code, I thought this was nice enough 😆 
EDIT: well, not always fail, but prove more effort to design than it was worth.

**Comment by VikingScientist** [comment]
This is ready to be reviewed and merged.

**Comment by akva2** [comment]
seems @TheBB has looked at it and didn't have much comments so i'm going to put my head on the block.

**Comment by TheBB** [comment]
Yes no problem.

**Pull request merged**